src/exojax/spec/moldb.py

```diff
@@ -169,7 +184,13 @@ def __init__(self, path, nurange=[-np.inf, np.inf], margin=0.0, crit=-np.inf, bk
                     cdt = (trans.nu_lines > self.nurange[0]-self.margin) \
                         * (trans.nu_lines < self.nurange[1]+self.margin)
                     if not np.isneginf(self.crit):
-                        cdt = cdt * (trans.Sij0 > self.crit)
+                        if not '_elower' in trans:
+                            print('The hdf5 file for the transition file was created using the old version of exojax. Remove', trans_file.with_suffix('.hdf5'), 'and try again.')
```

Comment:
The hdf5 file for the transition file was created using the old version of exojax<1.1.